tools/internal_ci/linux/grpc_bazel_on_foundry_opt.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
```

Comment:
I don't think this file has executable permissions.

Reply:
Copied from tools/jenkins/run_bazel_full.sh.

Reply:
I meant like running `chmod +x tools/internal_ci/linux/grpc_bazel_on_foundry_opt.sh`. GitHub is showing this file as not executable.

Reply:
Discussed offline. It works as is for the experiments. Will fix if something breaks.